tests/foreman/ui/test_packages.py

```diff
@@ -123,3 +131,68 @@ def test_positive_check_package_details(self):
                     ['Build Time', '1331831364'],
                 ]
             )
+
+
+class RHPackagesTestCase(UITestCase):
+    """Implement tests for packages via UI"""
+
+    @classmethod
+    @skip_if_not_set('fake_manifest')
+    def setUpClass(cls):
+        super(RHPackagesTestCase, cls).setUpClass()
+        cls.organization = entities.Organization().create()
+        with manifests.clone() as manifest:
+            upload_manifest(cls.organization.id, manifest.content)
+        repo_id = enable_rhrepo_and_fetchid(
+            basearch='x86_64',
+            org_id=cls.organization.id,
+            product=PRDS['rhel'],
+            repo=REPOS['rhst7']['name'],
+            reposet=REPOSET['rhst7'],
+            releasever=None,
+        )
+        entities.Repository(id=repo_id).sync()
+
+    @tier3
```

Comment:
E.g. api content view module contains:
7 tier2 tests [here](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_contentview.py#L1046), 4 [here](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_contentview.py#L1497) and 4 [here](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_contentview.py#L1629).
Test for syncing your specific repo is marked as tier2 too: https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_repository.py#L913
UI test which exercises different functionality but uses similar rh repo is marked as tier2 too: https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_activationkey.py#L964
Basically every rh repo sync test i see is marked as tier2, can't find both tier1 or tier3. Lots of tests which upload manifest are marked as tier1 but with real repo synchronization it's always tier2. No idea where did you get tier3, the only tests rh repo sync marked as tier3 i can see are the ones which work with errata later (and real hosts) or docker registries etc.